core/templates/dev/head/player/ConversationSkinDirective.js

```diff
@@ -441,11 +441,15 @@ oppia.directive('conversationSkin', [function() {
             $rootScope.loadingMessage = '';
             $scope.hasFullyLoaded = true;
 
-            // If the exploration is embedded, use as site language the
-            // exploration language.
+            // If the exploration is embedded, use the exploration language
+            // as site language. If the exploration language is not supported
+            // as site language, English is used as default.
             var siteLanguage = oppiaPlayerService.getExplorationLanguageCode();
-            if ($window.GLOBALS.SUPPORTED_SITE_LANGUAGES && $scope.isIframed) {
+            if ($window.GLOBALS.SUPPORTED_SITE_LANGUAGES[siteLanguage] &&
```

Comment:
First, it was an error, what I actually wanted to check is $window.GLOBALS.SUPPORTED_SITE_LANGUAGES[siteLanguage], to see if the language selected is supported as site language.
Second, $window.GLOBALS.SUPPORTED_SITE_LANGUAGES is set in the base.html file. All embedded explorations inherit from there, so it should always be present. Do you think we should check anyway?